backend/src/main/java/com/makro/model/SimpleTime.java

```diff
@@ -0,0 +1,29 @@
+package com.makro.model;
+
+public class SimpleTime {
+    private String username;
+    private float time;
+
+    public SimpleTime(String username, float time) {
+        this.username = username;
+        this.time = time;
+    }
+
+    public SimpleTime withUsername(String username) {
+        this.username = username;
+        return this;
+    }
+
+    public SimpleTime withTime(float time) {
+        this.time = time;
+        return this;
+    }
+
+    public String getUsername() {
+        return username;
+    }
+
+    public float getTime() {
+        return time;
+    }
```

Comment:
ja to bym lomboka zainstalował do tego

Reply:
What is lombok? This is only poc version, I think that we can think about improvements in following PRs.

Reply:
https://projectlombok.org/